### PR TITLE
[ISSUE #4201] Do some code optimization in PrometheusGrpcExporter and PrometheusExporterUtils

### PR DESCRIPTION
**Pull request by Sourav-Kumar-Panda**

### CODE REVIEW

**Comment by Pil0tXia** [comment]
Please link your PR with the relevant issue.

**Comment by Pil0tXia** [comment]
It seems the Task `b` in the issue was not accomplished in this PR.

**Comment by Sourav-Kumar-Panda** [comment]
> Please link your PR with the relevant issue.

While I was working on this issue, it was there but now this issue is closed.

**Comment by Sourav-Kumar-Panda** [comment]
> It seems the Task `b` in the issue was not accomplished in this PR.

Done with Task 'b'

File: eventmesh-metrics-plugin/eventmesh-metrics-prometheus/src/main/java/org/apache/eventmesh/metrics/prometheus/metrics/PrometheusGrpcExporter.java [hunk 1]
```diff
@@ -36,30 +37,29 @@
...
+                    { "server.tps", "get size of retry queue.", GrpcSummaryMetrics::getClient2EventMeshTPS },
+                    { "client.tps", "get tps of eventMesh to mq.", GrpcSummaryMetrics::getEventMesh2ClientTPS },
+                    { "mq.provider.tps", "get tps of eventMesh to mq.", GrpcSummaryMetrics::getEventMesh2MqTPS },
+                    { "mq.consumer.tps", "get tps of eventMesh to mq.", GrpcSummaryMetrics::getMq2EventMeshTPS }
             }
-        };
-    }
+    ).collect(Collectors.toMap(
+            data -> join((String)data[0], (String)data[1]),
+            data -> (Function<GrpcSummaryMetrics, Number>)data[2]
```
[the remark this answers is not in the document]
Comment (Sourav-Kumar-Panda) [reply eventmesh-metrics-plugin/eventmesh-metrics-prometheus/src/main/java/org/apache/eventmesh/metrics/prometheus/metrics/PrometheusGrpcExporter.java:51]: I use another way to initialize this instance, but the result is same.

File: eventmesh-metrics-plugin/eventmesh-metrics-prometheus/src/main/java/org/apache/eventmesh/metrics/prometheus/metrics/PrometheusGrpcExporter.java [hunk 2]
```diff
@@ -36,30 +37,29 @@
...
+    ));
 
     public static void export(final String meterName, final GrpcSummaryMetrics summaryMetrics) {
         final Meter meter = GlobalMeterProvider.getMeter(meterName);
 
-        paramPairs.forEach((metricInfo, getMetric) ->
-            observeOfValue(meter, METRICS_GRPC_PREFIX + metricInfo[0], metricInfo[1], GRPC, summaryMetrics, getMetric));
+        paramPairs.forEach((metricName, getMetric) ->
+                observeOfValue(meter, METRICS_GRPC_PREFIX + metricName, "get " + metricName, GRPC, summaryMetrics, getMetric));
```
[the remark this answers is not in the document]
Comment (Sourav-Kumar-Panda) [reply eventmesh-metrics-plugin/eventmesh-metrics-prometheus/src/main/java/org/apache/eventmesh/metrics/prometheus/metrics/PrometheusGrpcExporter.java:58]: I don't know how to solve this, could you please guide me?

File: eventmesh-metrics-plugin/eventmesh-metrics-prometheus/src/main/java/org/apache/eventmesh/metrics/prometheus/metrics/PrometheusGrpcExporter.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sourav-Kumar-Panda) [reply eventmesh-metrics-plugin/eventmesh-metrics-prometheus/src/main/java/org/apache/eventmesh/metrics/prometheus/metrics/PrometheusGrpcExporter.java:51]: I think it's working, just try it once.

**Pull request closed without merging**